.github/workflows/storage-layout-update.yml

```diff
@@ -0,0 +1,43 @@
+name: Update .storage-layout
+
+on:
+  workflow_dispatch:
+
+jobs:
+  assert:
+    runs-on: ubuntu-latest
+    name: Update storage layout
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      
+      - name: Setup node.js version
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          cache: 'yarn'
+
+      - name: Install yarn deps
+        run: yarn install --immutable
+
+      - name: Generate new .storage-layout
+        uses: lidofinance/storage-layout-action@v1
+        with:
+          mode: generate
+          src-folder: ./contracts
+          ignore-folders: "{test_helpers,template,mocks}"
+
+      - name: Check for changes
+        id: changes
+        continue-on-error: true
+        run: .github/assert-git-changes.py
```

Comment:
I propose to set an env or pass additional flag to know exactly what are we looking for

Reply:
Added